cooperative_lanechange/src/cooperative_lanechange.cpp

```diff
@@ -216,6 +220,9 @@ namespace cooperative_lanechange
         //get subject vehicle info
         lanelet::BasicPoint2d veh_pos(req.vehicle_state.X_pos_global, req.vehicle_state.Y_pos_global);
         double current_downtrack = wm_->routeTrackPos(veh_pos).downtrack;
+        if(current_downtrack < maneuver_plan[0].lane_change_maneuver.start_dist - 5.0){
```

Comment:
Should this be using that `starting_downtrack_range` parameter instead of a fixed 5.0?